scripts/compute_speed_table.py

```
@@ -596,6 +778,90 @@ def apply_to_rust(results: list[Calibration]) -> int:
+    """segment_speed_table.rs の生成ブロックを更新する。
+
+    今回の実行で駅間サンプルを較正できた路線のエントリは新しい結果で置き換え
+    (閾値未満になったペアは削除)、較正対象にならなかった路線の既存エントリは
+    そのまま残す。
+    """
+    with open(SEGMENT_TABLE_RS, encoding="utf-8") as f:
+        src = f.read()
+    start = src.find(BEGIN_MARK)
+    end = src.find(END_MARK)
+    if start < 0 or end < 0 or end < start:
+        raise SystemExit(f"{SEGMENT_TABLE_RS} に生成ブロックマーカーが見つかりません")
```

Ruff (0.15.20)

[warning] Avoid specifying long messages outside the exception class

(TRY003)